platform/app/src/routes/Mode/Mode.tsx

```diff
@@ -82,24 +125,7 @@ function defaultRouteInit(
   // until we run the hanging protocol matching service.
 
   Promise.allSettled(allRetrieves).then(() => {
```

Comment:
Just change this to select the promises to wait for before applying the hanging protocol - I know this is the series retrieves, not the display sets, but I think it is reasonable to apply here.

Reply:
Need to understand better that... I am not sure about that

Reply:
Changed as discussed in the meeting